MoviesApp/Core/AssemblyMovie.swift

```diff
@@ -26,7 +32,19 @@ final class AssemblyMovie: AssemblyMovieProtocol {
     func createMovieDetailModule(movieId: Int?, router: RouterMoviProtocol) -> UIViewController {
         let view = MovieDetailViewController()
         let moviAPIService = MovieAPIService()
-        let presenter = MovieDetailViewPresenter(view: view, moviAPIService: moviAPIService, movieID: movieId)
+        let entityMapper = MovieInfoEntityMapper()
+        let contextProvider = DBContextProvider()
+        let imageAPIService = ImageAPIService()
+        let cacheImageService = CacheImageService()
+        let loadImageProxy = LoadImageProxy(service: imageAPIService, cacheImageService: cacheImageService)
+        let coreDataRepository = CoreDataRepository(contextProvider: contextProvider, entityMapper: entityMapper)
```

Comment:
```suggestion
         let entityMapper = MovieInfoEntityMapper()
         let contextProvider = DBContextProvider()
        let coreDataRepository = CoreDataRepository(contextProvider: contextProvider, entityMapper: entityMapper)
```
это все не нужно модулю инкапсулируем в dataStoradgeService как я писал в тз!